modules/services/redshift-gammastep/lib/options.nix

```diff
@@ -75,26 +107,6 @@ in {
       };
     };
 
-    brightness = {
-      day = mkOption {
-        type = types.str;
-        default = "1";
-        description = ''
-          Screen brightness to apply during the day,
-          between <literal>0.1</literal> and <literal>1.0</literal>.
-        '';
-      };
-
-      night = mkOption {
-        type = types.str;
-        default = "1";
-        description = ''
-          Screen brightness to apply during the night,
-          between <literal>0.1</literal> and <literal>1.0</literal>.
-        '';
-      };
-    };
```

Comment:
Removing it since RFC42 only recommends to support most used options. Both brightness and gamma are probably equally used (and most people don't set it as default), so I removed it.

It still is supported via a rename, but will show a warning message.

Reply:
I don't understand why these options were removed. RFC42 is for reducing maintenance burden, and I don't think having these options exposed makes it difficult to maintain this module.

On the contrary, it just breaks peoples configs downstream.

Reply:
They're not completely removed, thanks to `mkRenamedOption` it should still work (but it will show a warning).

Reply:
Anyway, this is completely off-topic. This PR was already merged some time ago, if you have some complains open an issue or a PR.

Reply:
If that's how the `mkRenamedOption` is suppose to be used, then there might be something wrong with it. The change causes an error to be thrown if including the brightness option.

```
error: --- ThrownError --------------------------------------------------------------------------------------------------------------------nix
The option `home-manager.users.*.services.redshift.brightness' does not exist. Definition values:
- In `/nix/store/hci9dfhymzjdh46wbjn58c93myalk7bj-source/home-manager/desktop.nix':
    {
      day = "1";
      night = "0.75";
    }
(use '--show-trace' to show detailed location information)
```

I'll open up a new PR to fix this.